src/main/kotlin/no/nav/syfo/senoppfolging/SenOppfolgingControllerV1.kt

```diff
@@ -70,7 +70,8 @@ class SenOppfolgingControllerV1(
         )
         log.info(
             "veilarbregistrering type [${startRegistration.registreringType},${startRegistration.formidlingsgruppe}," +
```

Comment:
Mulig du vil ha med en prefix som sier hva det er? Ala
"""
                Veilarbregistrering type:${startRegistration.registreringType},
                |Formidlingsgruppe: ${startRegistration.formidlingsgruppe},
                |Servicegruppe: ${startRegistration.servicegruppe},
                |Rettighetsgruppe: ${startRegistration.rettighetsgruppe},
                |Sykmeldt: $isSykmeldt,
                |Responsestatus: ${response.toResponseStatus()}
            """.trimMargin(),

Reply:
Ja, egentlig, men har laget det sånn for at det skal være enkelt å eksportere til excel

Reply:
åja